rollup.config.js

```diff
@@ -0,0 +1,9 @@
+import babel from 'rollup-plugin-babel';
```

Comment:
babel is such an appropriate name for this project, in two years from now nobody will be able to understand each other :P